lib/dialects/sqlite/query-interface.js

```diff
@@ -29,14 +29,16 @@ var removeColumn = function(tableName, attributeName, options) {
   var self = this;
   options = options || {};
 
-  return this.describeTable(tableName).then(function(fields) {
+  return this.describeTable(tableName, options).then(function(fields) {
     delete fields[attributeName];
 
     var sql = self.QueryGenerator.removeColumnQuery(tableName, fields)
       , subQueries = sql.split(';').filter(function(q) { return q !== ''; });
 
+    var rawOptions = Utils._.clone(options);
+    rawOptions.raw = true;
     return Promise.each(subQueries, function(subQuery) {
-      return self.sequelize.query(subQuery + ';', { raw: true, logging: options.logging });
+      return self.sequelize.query(subQuery + ';', rawOptions);
```

Comment:
Ah I was looking for something like that, thanks! As long as it does not touch the original options object :)